src/traces/scatter3d/convert.js

```diff
@@ -343,7 +343,7 @@ proto.update = function(data) {
     this.dataPoints = options.position;
 
     lineOptions = {
-        gl: gl,
+        gl: this.scene.glplot.gl,
```

Comment:
Why is `this.scene.glplot.gl` different than `gl` in this scope?

Reply:
The `gl` object there has rather plain structure and didn't have pointers to `pixelRatio` attribute and other methods. Passing `glplot.gl` here is similar to the way is passed for other traces e.g. `mesh3d` & `surface3d`.

Reply:
Thanks!